src/containerapp/azext_containerapp/_clients.py

```diff
@@ -556,7 +556,7 @@ def create_or_update(cls, cmd, resource_group_name, name, github_action_envelope
     @classmethod
     def show(cls, cmd, resource_group_name, name):
         management_hostname = cmd.cli_ctx.cloud.endpoints.resource_manager
-        api_version = NEW_API_VERSION
+        api_version = "2022-03-01"
```

Comment:
Same as above

Reply:
@StrawnSC we should probably name these preview-version & stable-version, so the swap when we need to do is easy. "NEW_API_VERSION"  name here is misleading, since this is the not the new version.

Reply:
Just resolved this @panchagnula @zhoxing-ms